torch/distributions/geometric.py

```diff
@@ -0,0 +1,66 @@
+from numbers import Number
+
+import torch
+from torch.distributions import constraints
+from torch.distributions.distribution import Distribution
+from torch.distributions.utils import broadcast_all, probs_to_logits, logits_to_probs, lazy_property, _finfo
+from torch.nn.functional import binary_cross_entropy_with_logits
+
+
+class Geometric(Distribution):
+    r"""
+    Creates a Geometric distribution parameterized by `probs`. It represents the probability that in
+    k + 1 Bernoulli trials, the first k trials failed, before seeing a success
```

Comment:
nit: period.

Could you also mention that the mean is `(1-p)/p`? I always forget whether Geometric is parameterized by `p` or `1-p` and it would be nice to remind forgetful users like me in which direction `p` is oriented.